internal/hooks/v0hooks/manager.go

```diff
@@ -382,6 +269,168 @@ func (a *API) runHook(r *http.Request, conn *storage.Connection, hookConfig conf
 	return response, nil
 }
 
+func (o *Manager) runPostgresHook(
```

Comment:
hard to tell from the diff but the changes here just move these hook functions to be a struct method of `Manager` and under a different package right? (no longer under `api` but moved to `v0hooks`?)

Reply:
That's correct, the main goal was moving them out of internal/api.